Scottish-Tech-Army/Volunteer-app
# SVA-316 reformat links in the project descriptions

- [Description](#description)
  - [Type of change](#type-of-change)
- [Testing locally](#testing-locally)
- [Checklist](#checklist)

# Description

- In the description some urls and emails are wrapped in square brackets. 
- These code updates finds them, removes the unwanted characters from the weblinks, and makes them clickable hyperlinks. 
- Email addresses are removed (sensitive and unnecessary).  

## Type of change

Please delete options that are not relevant

- [] New Feature (non-breaking change which adds functionality)
- [] Breaking change(fix or feature that would cause existing functionality to not work as expected)
- [] This change requires a documentation update

# Testing locally

- run the app locally - start the api with `npm start --reset-cache` 
- check the links no longer have unwanted characters.

# Checklist

- [] I have made corresponding changes to the documentation (if required)
- [] I have addressed accessibility, if needed
- [] I have checked the app in dark mode, if making front-end design changes
